Cargo.toml

```
@@ -20,4 +20,9 @@ near-primitives = "0.17.0"
 near-jsonrpc-client = "0.6.0"
 near-jsonrpc-primitives = "0.17.0"
 
-interactive-clap = "0.2.6"
+near-token = { version = "0.1.0", features = [
+    "serde",
+    "borsh",
+    "schemars",
+    "interactive-clap",
+] }
```

I don't see any use of those features, so you should be able to safely skip adding those:

```suggestion
near-token = "0.1.0"
```